examples/interactive.jl

```diff
@@ -752,13 +683,14 @@ end
                 contour(volume, alpha = 0.1, levels = 10)
             )
         )
-
         # Do not execute beyond this point!
-
         RecordEvents(scene, @replace_with_a_path)
     end
 
-    @cell "Lighting" ["3d", lighting, ambient, diffuse, specular, shininess, lightposition, interaction] begin
+end
+
+@block FredericFreyer ["3d"] begin
+    @cell "Lighting" [lighting, ambient, diffuse, specular, shininess, lightposition, interaction] begin
```

Comment:
Let's add a lighting tag to the list of allowed tags as well!  I think that was somewhere in `MakieGallery.gallery_from_recordings`.